specification/elasticsan/resource-manager/Microsoft.ElasticSan/ElasticSan/main.tsp

```diff
@@ -39,9 +39,16 @@ namespace Microsoft.ElasticSan;
  * The available API versions.
  */
 enum Versions {
+  /**
+   * The 2025-09-01 stable API version.
+   */
+  @armCommonTypesVersion(Azure.ResourceManager.CommonTypes.Versions.v3)
```

Comment:
Versions should be *in order*  having the versions in this order makes an `@added` in the preview remove a type from the stable version.

Also, why are we adding an old preview version to the typespec?  You should only add new versions going forward